src/utils/upload/processUpload.js

```diff
@@ -22,6 +22,17 @@ const putInS3 = async (projectUuid, loadedFileData, dispatch, sampleUuid, fileNa
   const signedUrlResponse = await fetchAPI(
     url,
     {
+      metadata,
+      progressCallback(progress) {
+        const percentProgress = Math.round((progress.loaded / progress.total) * 100);
+        console.log('Calling from putInS3');
+        dispatch(updateSampleFile(sampleUuid, fileName, {
+          upload: {
+            status: UploadStatus.UPLOADING,
+            progress: percentProgress ?? 0,
+          },
+        }));
+      },
```

Comment:
I don't have any memory of changing that, I have to check where is this coming from